libexec/pyenv-init

```diff
@@ -244,7 +237,7 @@ function print_env() {
 }
 
 function print_completion() {
-  completion="${root}/completions/pyenv.${shell}"
+  completion="${PYENV_ROOT}/completions/pyenv.${shell}"
```

Comment:
This must be `$root` -- this is executable root, not data root. The distinction that I was speaking about earlier.

Reply:
Rather accidentally when writing documentation in the advanced section, I actually found that pyenv indeed assumes that completions shall be located directly under $PYENV_ROOT, so assuming that completions shall not be under $PYENV_ROOT should not be a problem ? 😇

This is from README.md
```
3. **Installs autocompletion.** This is entirely optional but pretty
   useful. Sourcing `$(pyenv root)/completions/pyenv.bash` will set that
   up. There are also completions for Zsh and Fish.
```

Reply:
> Rather accidentally when writing documentation in the advanced section

See? I knew what I was doing when I directed you there! :innocent: 

> so assuming that completions shall not be under $PYENV_ROOT should not be a problem ? 😇

Not a problem at all, they are not:

```
% find $(brew --prefix) -name pyenv.bash
/opt/homebrew-custom/Cellar/pyenv/2.4.22/completions/pyenv.bash
```

So it's your fortunate find that needs fixing!

Reply:
💀

Reply:
I guess I'll install homebrew just to understand how the folder structure works there.
Of course, we can just revert to:

```
function print_completion() {
  completion="${0%/*}/../completions/pyenv.${shell}"
  if [ -r "$completion" ]; then
    echo "source '$completion'"
  fi
}
```

![image](https://github.com/user-attachments/assets/99756591-334f-4df1-962b-fdb0b0fb14da)
![image](https://github.com/user-attachments/assets/99edc683-aeec-4d12-aeeb-d871016e3824)

Reply:
> to understand how the folder structure works there

Basically, whatever is part of the repo is code, whatever is created by the installation is data. `plugins` can be in both places IIRC.